frontend/src/bundles/resume/pages/create-resume/create-resume.tsx

```diff
@@ -0,0 +1,41 @@
+import { Header, RegularButton } from '~/bundles/common/components/components';
+import { ButtonVariant } from '~/bundles/common/enums/enums';
+import { useResumes } from '~/bundles/common/hooks/hooks';
+import { OnlineEditorTabsHandler } from '~/bundles/cv-editor/components/components';
+import { TemplateEditor } from '~/bundles/edit-template/components/template-editor/template-editor';
+
+import styles from './styles.module.scss';
+
+const CreateResume: React.FC = () => {
+    const { templateSettings, createResume, setTemplateSettingsMockData } =
+        useResumes();
+
+    return (
+        <>
+            <Header />
+            <div className={styles.create__resume}>
+                {templateSettings && (
+                    <>
+                        <OnlineEditorTabsHandler
+                            tabs={templateSettings.containers}
+                            isCreate
+                        />
+                        <TemplateEditor
```

Comment:
consider creating a separate component for preview so it doenst have drag and drop functions

Reply:
Yeah, good notice, was going to do so